client/src/pages/onboarding/components/EmptyStarterCredential.tsx

```diff
@@ -0,0 +1,32 @@
+import { motion } from 'framer-motion'
+import React from 'react'
+
+import { fadeX } from '../../../FramerAnimations'
+import { StateIndicator } from '../../../components/StateIndicator'
+import { prependApiUrl } from '../../../utils/Url'
+
+export const EmptyStarterCredential: React.FC = () => {
```

Comment:
Is there a difference between this and the other empty credential?

Reply:
Slight difference in the way you used them in the AcceptCredential component, hence I can't use EmptyCredential in the AcceptCredential component

Reply:
Hmm yea i see. I think it would be nice if we can merge these as they are basically displaying the same thing right? Would you be able to do that?

Reply:
Okay. Should be doable. On it.